src/types/db.ts

```diff
@@ -41,6 +42,7 @@ export interface Database {
     readonly url: string;
     tables: Tables
     dbname: string;
+    logger: typeof consola
```

Comment:
logger to have an interface to be abstracted, should not be a param at higher levels